src/main/java/johnny/Parser.java

```diff
@@ -0,0 +1,111 @@
+package johnny;
+
+import java.util.ArrayList;
+
+public class Parser {
+
+    private String input;
+
+    public static String TERMINATE = "TERMINATE";
+    public static String PRINT_LIST = "PRINT_LIST";
+    public static String MARK = "MARK";
+    public static String UNMARK = "UNMARK";
+    public static String DELETE = "DELETE";
+    public static String ADD_TODO = "ADD_TODO";
+    public static String ADD_DEADLINE = "ADD_DEADLINE";
+    public static String ADD_EVENT = "ADD_EVENT";
+    public static String FIND_EVENT = "FIND_EVENT";
+
+
+    public Parser(String userInput) {
+        this.input = userInput;
+    }
+
+    private static boolean isNumeric(String str) {
+        try {
+            Integer.parseInt(str);
+            return true;
+        } catch(NumberFormatException e){
```

Comment:
```suggestion
        } catch (NumberFormatException e) {
```
I agree with Kidsnd274 too! In addition, there should be a whitespace between ( and {